### DISCUSSION TITLE
Alist一启动C盘就占用大量空间

### DISCUSSION THREAD

**Discussion by nothing0here** [AlistGo/alist#7323] [Q&A]
有个问题困扰我很久了，之前曾经复制大文件失败过，现在每次启动alist，C盘空间就会在五六分钟内增加24G的空间占用，然后通过各种方式终于在appdata\Roaming\ci.nn.alist\temp路径下找到一个文件名为file-4145561464，24G大小的文件，无法被删除，停止alist服务后这个文件会自动删除，但是重新启动alist服务后又会自动生成。尝试删除配置文件夹重新安装alist_desktop依旧无法解决，求大佬搭救。
![image](https://github.com/user-attachments/assets/ffd5b5a2-5ad5-45d8-a6f8-b6585e92c304)
![image](https://github.com/user-attachments/assets/b852752b-d7f1-4c7c-9ef2-d91abe619710)
![image](https://github.com/user-attachments/assets/f6e5c867-50d6-4554-a0b1-b092507f1c8f)

**Comment by snugduahn** [comment]
你这个好像是在C盘那个路径下生成缓存文件。实在不行，把ALIST卸载了重新安装一下吧。
